components/BusinessCard.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <v-card class="business-card">
+    <!-- City -->
+    <v-chip class="px-2 business-city">
+      <span>{{ city }}</span>
+    </v-chip>
+    <!-- Image -->
+    <div class="business-image">
+      <img
+        :alt="business.title + ' image'"
+        :src="
+          business.preview_image ||
+          require('@/static/images/fallback_preview.png')
+        "
+      />
+    </div>
+    <!-- Content -->
+    <div class="business-content card-content flex-column elevation-3">
+      <!-- Logo -->
+      <business-logo
+        :logo-info="{ logoFallback: business.title, logoPath: business.logo }"
+        :style="logoSize"
+      />
+      <!-- Like -->
+      <v-btn
+        class="like-btn"
+        color="white"
+        fab
+        @click="TOGGLE_LIKE(business.id)"
+      >
+        <v-icon large color="black">
+          {{ business.is_liked ? 'mdi-heart' : 'mdi-heart-outline' }}
+        </v-icon>
+      </v-btn>
+      <!-- Title -->
+      <div class="pt-3 px-3 business-title headline-sm text-truncate">
+        {{ business.title }}
+      </div>
+      <!-- Tags -->
+      <div v-if="business.tags.length" class="px-3 d-flex align-center">
+        <v-chip class="elevation-1 mr-3 white">
+          <span class="text-truncate">{{ business.tags[0] }}</span>
+        </v-chip>
+        <span v-if="business.tags.length > 1" class="mb-1 tags-link">
+          +{{ business.tags.length - 1 }} weitere
+        </span>
+      </div>
+      <!-- Description -->
+      <div :class="business.tags.length ? 'pb-3' : 'pb-5'" class="pt-1 px-3">
+        <div class="business-description">
+          {{ business.description }}
+        </div>
+      </div>
+    </div>
+  </v-card>
+</template>
+
+<script>
+import { mapMutations } from 'vuex'
+import BusinessLogo from './BusinessLogo.vue'
+
+export default {
+  name: 'BusinessCard',
+
+  components: {
+    BusinessLogo,
+  },
+
+  props: {
+    city: {
+      type: String,
+      default: '',
+    },
+    business: {
+      type: Object,
+      default: () => ({
+        logo: '',
+        title: '',
+        tags: [],
+        description: '',
+        is_liked: false,
+      }),
+    },
+  },
+
+  data: () => ({
+    logoSize: {
```

Comment:
I would move the width into a prop of business logo and the other styles into a css class.
Size as prop so the component could load different sized images based on with and height.